src/org/intellij/erlang/utils/ErlangModulesUtil.java

```diff
@@ -93,21 +93,36 @@ private static ErlangModule doGetErlangModule(@NotNull Project project,
   public static Collection<ErlangFile> getErlangModules(@NotNull Project project) {
     HashSet<ErlangFile> erlangModules = new HashSet<ErlangFile>();
     for (Module module : ModuleManager.getInstance(project).getModules()) {
-      addErlangModules(module, false, erlangModules);
+      addErlangFiles(module, false, erlangModules, ErlangFileType.MODULE);
     }
     return erlangModules;
   }
 
   @NotNull
   public static Collection<ErlangFile> getErlangModules(@NotNull Module module, boolean onlyTestModules) {
-    return addErlangModules(module, onlyTestModules, new HashSet<ErlangFile>());
+    return addErlangFiles(module, onlyTestModules, new HashSet<ErlangFile>(), ErlangFileType.MODULE);
   }
 
   @NotNull
-  private static Collection<ErlangFile> addErlangModules(@NotNull Module module, boolean onlyTestModules, @NotNull Collection<ErlangFile> erlangModules) {
+  public static Collection<VirtualFile> getVirtualFilesForErlangHeaders(@NotNull Module module,
+                                                                        boolean onlyTestModules) {
+    return addFiles(module, onlyTestModules, ContainerUtil.<VirtualFile>newHashSet(), ErlangFileType.HEADER);
+  }
+
+  @NotNull
+  public static Collection<VirtualFile> getVirtualFilesForErlangModules(@NotNull Module module,
```

Comment:
I believe, these method names can be made shorter without loosing information - why not name them getErlangModuleFiles and ErlangHeaderFiles instead?